docs/paper/dev/misc/debugging.md

```diff
@@ -0,0 +1,76 @@
+# Debugging your plugin
+
+Debugging your plugin is vital to being able to fix bugs and issues in your plugin. This page will cover some of the most common debugging techniques.
+
+### Randomly printing to the console
+
+One of the most common debugging techniques is to print to the console. This is likely something you've done before as its very simple. 
+It has a few downsides though. It can be hard to find the print statements in the console, and it can be hard to remove them all when you're done debugging.
+
+When debugging, you can use `System.out.println("");` to print to the console. It is recommended to use your plugin's logger instead though, 
+as it will be easier to know which plugin the log has come from. This can be done simply with:
+
+```java
+plugin.getLogger().warning("SuperDuperBad Thing has happened");
```

Comment:
I think we just want to point to `Plugin#getComponentLogger` or `Plugin#getSLF4JLogger` instead of the java.util.logging.Logger